tests/System.Text.JsonLab.Tests/JsonWriterTests.cs

```diff
@@ -17,15 +19,17 @@ public class JsonWriterTests
         public void WriteJsonUtf8()
         {
             var formatter = new ArrayFormatter(1024, SymbolTable.InvariantUtf8);
-            var json = new JsonWriter(formatter, true, prettyPrint: false);
+            var bufferWriter = new BufferWriter<IBufferWriter<byte>>(formatter);
```

Comment:
```csharp
var bufferWriter = BufferWriter.Create(formatter);
var json = JsonWriter.Create(formatter, isUtf8: true, prettyPrint: false);
```

Reply:
Did you mean to pass-in bufferWriter to JsonWriter.Create? If not, then why would we need to create bufferWriter at all?

Reply:
Yes, edited :)